docs/rules/no-loose-assertions.md

```diff
@@ -0,0 +1,87 @@
+# Forbid the use of assert.equal/assert.ok/assert.notStrictEqual/assert.notOk (no-loose-assertions)
+
+The `assert.equal` assertion method in QUnit uses loose equality comparison. In a project which favors strict equality comparison, it is better to use `assert.strictEqual` for scalar values and either `assert.deepEqual` or `assert.propEqual` for more complex objects.
```

Comment:
Could you please add a sentence saying that `assert.notEqual` has similar issues? Up to you how detailed you want to make it.

Reply:
Done. Let me know if this looks better :-).